package.json

```diff
@@ -34,8 +34,12 @@
     "@typescript-eslint/parser": "^5.57.1",
     "@vitejs/plugin-react": "^4.0.0",
     "eslint": "^8.38.0",
+    "eslint-config-airbnb-typescript": "^17.1.0",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-react-refresh": "^0.3.4",
```

Comment:
Same comment as here: https://github.com/iExecBlockchainComputing/dataprotector-sandbox/pull/8#issuecomment-1775492838

eslint-plugin-react-refresh is referenced as a dev dependency but never used. Either we use it, or let's clean the dependency? https://github.com/ArnaudBarre/eslint-plugin-react-refresh#installation